R/teal_data-get_code.R

```diff
@@ -74,14 +74,16 @@
 #' @param object (`teal_data`)
 #' @param datanames `r lifecycle::badge("experimental")` (`character`) vector of dataset names to return the code for.
```

Comment:
please update this

Reply:
thanks!
https://github.com/insightsengineering/teal.data/pull/343/commits/345f398b08f5847ee6392522406459d8836476e1